### PR TITLE
Ft/hero section

### PR DESCRIPTION
**Pull request by peninah98**
# Hero section Pull Request

- [x] Created here section content
- [x] Adding fonts 
- [x] Positioning
- [x] Added Picture
- [x] Positioned it so it would look like the Figma design

## Motivation and Context
The changes in this Pull Request create and render section that includes here section eye-catching content and image in the html index file

## How Has This Been Tested, and How are your changes run?
To run this file clone it [link](https://github.com/Nezerwa/Ojemba-VLM-Project/compare/ft/heroSection?expand=1)
Have nodejs  and npm installed
Run `npx tailwindcss -i ./src/input.css -o ./dist/output.css --watch`

## Screenshots 
image

### CODE REVIEW

**Review by ell-sie — COMMENTED** [review]
Great Job, very well executed you can merge!

**Review by ell-sie — APPROVED** [review]
Approved!

**Review by Nezerwa — APPROVED** [review]
Good job @peninah98  , your work is done it's time to merge!

File: index.html [hunk 1]
```diff
@@ -4,6 +4,8 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link href="/dist/output.css" rel="stylesheet">
+  <!-- <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&family=Roboto&display=swap" rel="stylesheet"> -->
```
Comment (alainmigabo) [inline index.html]: Could you please remove this comment

**Review by alainmigabo — CHANGES_REQUESTED** [review]

**Comment by alainmigabo** [comment]
First Resolve your conflicts

**Review by Nezerwa — CHANGES_REQUESTED** [review]
well done , but try to also fix merge conflicts

**Comment by ell-sie** [comment]
> First Resolve your conflicts

I have finished resolving them.

**Comment by ell-sie** [comment]
> well done , but try to also fix merge conflicts

Done

**Review by Nezerwa — APPROVED** [review]
good job on the conflict have resolved

**Review by alainmigabo — APPROVED** [review]

**Review by Nezerwa — APPROVED** [review]
well  done @ell-sie  you can merge

**Comment by ell-sie** [comment]
Confirm merge

**Pull request merged**